ansible/roles/satellite_master/templates/satellite-master.j2

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Copyright 2015 TWO SIGMA OPEN SOURCE, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cd {{ satellite_install_dir }}
+export PATH=$PATH:.
```

Comment:
Actually, for satellite-master, $PATH doesn't need to be manipulated.  So I'll remove that from here. 

With satellite-slave, we need to ensure that $PATH contains satellite-recipes.